mmv1/templates/terraform/pre_create/vertex_ai_models.tmpl

```diff
@@ -0,0 +1,9 @@
+var postRequestType string
+if _, ok := d.GetOk("model"); ok {
+	postRequestType = ":upload"
+}
```

Comment:
Is there a way you can write this pre_create in a way that's specific to create-via-copy and doesn't have anything related to upload?

Reply:
removed